docs/static/releasenotes.asciidoc

```diff
@@ -10,12 +10,12 @@ This section summarizes the changes in the following releases:
 [[logstash-6-0-0-beta1]]
 === Logstash 6.0.0-beta1 Release Notes
 
-Placeholder for beta1 release notes
-
-* Added new `logstash.yml` setting: `config.support_escapes`. When
-  enabled, Logstash will interpret escape sequences in strings in the pipeline
-  configuration.
-
+* Added new `logstash.yml` setting: `config.support_escapes`. When enabled, Logstash will interpret escape sequences in      strings in the pipeline configuration.
+* Breaking: The setting `config.reload.interval` has been changed to use time value strings such as `5m`, `10s`, etc for convenience. Previously, users had to convert this to a millisecond time value themselves.
+* Breaking: The list of plugins bundled with Logstash beta1 release has changed. Please consult the breaking changes document for a complete list.
```

Comment:
I wonder if you should say 6.0.0-beta 1 release (to be consistent with how the version will be referred to elsewhere).